tests/lib/prepare-project.sh

```diff
@@ -119,18 +121,15 @@ if [[ "$SPREAD_SYSTEM" == ubuntu-14.04-* ]]; then
     quiet apt-get install -y --force-yes apparmor libapparmor1 seccomp libseccomp2 systemd cgroup-lite util-linux
 fi
 
-quiet apt-get purge -y snapd
+distro_purge_package snapd
```

Comment:
A bit of a bike-shead but instead of building a pretend uniform map of packages across distributions (wait until we get to gentoo) how about just having a function like this:

```
install_package debian:snapd fedora:snapd

Which will use `ID` and `ID_LIKE` to install the subset of packages that match what is given on command line. Then we don't need to maintain any central mapping of any kind. The function can be made to fail if no packages match (say we add openSUSE) for better reliability.
```